base/dict.jl

```diff
@@ -776,20 +776,59 @@ struct ImmutableDict{K,V} <: AbstractDict{K,V}
 end
 
 """
-    ImmutableDict
+    ImmutableDict(key=>value, key=>value, ...)
```

Comment:
```suggestion
    ImmutableDict(key1=>value1, key2=>value2, ...)
```
I think this one got missed